src/product-card.css

```diff
@@ -0,0 +1,96 @@
+.product-card {
+  --product-card-width: 200px;
+
+  display: flex;
+  flex-direction: column;
+  width: var(--product-card-width);
+  flex-shrink: 0;
+  border: 1px solid #f3f3f3;
+  background-color: #fff;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+  padding: 0 calc(var(--product-card-width) * 0.08);
```

Comment:
the card should have top and bottom padding too

###top
![image](https://user-images.githubusercontent.com/63306833/173360730-8c40915e-71c7-4dbb-bbe7-3b947dba39cb.png)

###left and right
![image](https://user-images.githubusercontent.com/63306833/173360939-1d62306e-caf7-4321-a161-01f32fa2205d.png)


###bottom
![image](https://user-images.githubusercontent.com/63306833/173360782-8671a0ff-022a-4858-9a3b-cc2a6e7232de.png)